File: R/tests/testthat/test_bart_prospect_learning.R
context("Test bart_prospect_learning")
library(hBayesDM)

test_that("Test bart_prospect_learning", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(bart_prospect_learning(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})

File: Python/tests/test_bart_prospect_learning.py
import pytest

from hbayesdm.models import bart_prospect_learning


def test_bart_prospect_learning():
    _ = bart_prospect_learning(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()
